demo/dsa-accel-config-demo/Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN dnf install -y \
     wget autoconf automake libtool m4 diffutils file make dnf-utils patch \
     libuuid-devel json-c-devel
 
-ARG ACCEL_CONFIG_VERSION=v3.4.6.3
+ARG ACCEL_CONFIG_VERSION=v3.4.6.4
```

Comment:
this can go to a PR of it's own (with the same version updated to `intel-idxd-config-initcontainer`)

Reply:
@mythi done: #1038